etc/systemd/system/carbon-cache.service.d/serviceparam.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+LimitNOFILE=500000
```

Comment:
Do you have some reasoning behind this particular number?

Reply:
Increasing the Max no of open fds to 500000 to accomodate the connections when operating at scale. Have seen that when server is loaded, it takes time to close the connections and the open fds increases steadily. Given a higher number to make sure we are not running out of fds.